pkgs/desktops/gnome/extensions/extensionOverridesPatches/vitals_at_corecoding.com.patch

```diff
@@ -1,13 +1,17 @@
 diff --git a/sensors.js b/sensors.js
-index 39d175a..9815b77 100644
+index f310161..3926a17 100644
 --- a/sensors.js
 +++ b/sensors.js
-@@ -29,6 +29,8 @@ import * as FileModule from './helpers/file.js';
+@@ -30,9 +30,11 @@ import * as FileModule from './helpers/file.js';
  import { gettext as _ } from 'resource:///org/gnome/shell/extensions/extension.js';
  import NM from 'gi://NM';
-
-+imports.gi.GIRepository.Repository.prepend_search_path('@gtop_path@');
+ 
++imports.gi.GIRepository.Repository.dup_default().prepend_search_path('@gtop_path@');
 +
  let GTop, hasGTop = true;
  try {
-     ({default: GTop} = await import('gi://GTop'));
+-    ({default: GTop} = await import('gi://GTop'));
++    ({default: GTop} = await import('gi://GTop?version=2.0'));
```

Comment:
Do you think upstream would be interested in _this_ part of the change? Using an explicit version?